releasenotes/notes/cmake-python-preserve-lc-rpath-2e6b0f93d9ee9e10.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Ensure cmake always uses static linking for the datadog-agent dylib when building for macos to fix dlopen failures.
```

Comment:
Is this accurate? The setting as per the [cmake docs](https://cmake.org/cmake/help/latest/prop_tgt/INSTALL_RPATH_USE_LINK_PATH.html) achieves the following:
> INSTALL_RPATH_USE_LINK_PATH is a boolean that if set to True will append to the runtime search path (rpath) of installed binaries any directories outside the project that are in the linker search path or contain linked library files.

Reply:
Good point, this has nothing to do with static linking. Updated